src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -223,6 +223,34 @@ describe('ThreadView', () => {
       expect(JSON.parse(axiosMock.history.patch[axiosMock.history.patch.length - 1].data)).toMatchObject(data);
     }
 
+    it('should not show post footer', async () => {
+      Factory.resetAll();
+      await axiosMock.onGet(`${threadsApiUrl}${discussionPostId}/`)
+        .reply(200, Factory.build('thread', {
+          vote_count: 0,
+          following: false,
+          closed: false,
+          group_id: null,
+        }));
+      await executeThunk(fetchThread(discussionPostId), store.dispatch, store.getState);
+      renderComponent(discussionPostId);
+      expect(screen.queryByTestId('post-footer')).not.toBeInTheDocument();
+    });
+
+    it('should show post footer', async () => {
+      Factory.resetAll();
```

Comment:
don't need to pass Factory.resetAll(); because its already present in the beforeEach. It will automatically call before executing each test case, so you can remove this line